src/App.tsx

```diff
@@ -1,18 +1,39 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
-
+import React from 'react';
+import { Routes, Navigate, Route } from 'react-router-dom';
+import { Navbar } from './components/Nav/Navbar';
+import { PeoplePage } from './components/People/PeoplePage/PeoplePage';
+import { HomePage } from './components/HomePage/HomePage';
+import { NotFoundPage } from './components/NotFoundPage/NotFoundPage';
 import './App.scss';
 
-export const App = () => {
+export const App: React.FC = () => {
   return (
     <div data-cy="app">
       <Navbar />
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route
+              path="/"
```

Comment:
It's better to have some kind of variable that stores all route values in one place, enum, or just a const. It'll be very clear to see all existing routes in one place and also when in some other components you have to use a redirect or link you can import that variable and avoid using hardcoded values which usually cause bugs and makes refactoring harder.